src/screens/ContactScreen/elements/ItemUser.js

```diff
@@ -34,7 +36,7 @@ const ItemUser = ({photo, username, followed, onPress, userid}) => {
         style={[
           styles.containerCard,
           {
-            backgroundColor: isSelect ? COLORS.holytosca15percent : COLORS.white
+            backgroundColor: isSelect ? anonColorCondition : COLORS.white
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from a hardcoded color (`COLORS.holytosca15percent`) to a dynamic one (`anonColorCondition`) based on whether the user is anonymous or not is a good improvement for better UI customization.



<!-- This is an auto-generated comment by OSS CodeRabbit -->